### PR TITLE
Port DotaMatchRequest to netcore and put it under samples

### PR DESCRIPTION
**Pull request by xPaw**

### CODE REVIEW

File: Samples/Samples.sln [hunk 1]
```diff
@@ -1,25 +1,27 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26228.4
+VisualStudioVersion = 15.0.27004.2002
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sample1_Logon", "1.Logon\Sample1_Logon.csproj", "{CEF39496-576D-4A70-9A06-16112B84B79F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sample1_Logon", "1.Logon\Sample1_Logon.csproj", "{CEF39496-576D-4A70-9A06-16112B84B79F}"
```
Comment (xPaw) [inline Samples/Samples.sln]: Why does visual studio rewrite these?

Comment (VoiDeD) [reply Samples/Samples.sln]: Does it affect anything? It's probably just a project type guid or something.

Comment (yaakov-h) [reply Samples/Samples.sln]: https://github.com/dotnet/project-system/issues/1821

**Comment by xPaw** [comment]
I have fixed the line endings btw.

**Pull request merged**